packages/publish-app-layer/src/index.tsx

```diff
@@ -45,6 +45,14 @@ export default function () {
   const onCrossClickCb = useCallback(() => {
     setShowPopup(false);
   }, []);
+
+  // keyboard shortcut for run and build
+  const handleKeyPress = useCallback((event: { repeat: any; ctrlKey: boolean; key: string; }) => {
```

Comment:
Please provide a definite type to `repeat` or don't destructure here. You can destructure inside the function like 

```typescript
const { repeat, ctrlKey, key } = event;
```

Reply:
Not able to figure out the type of event.

Reply:
```
const handleKeyPress = useCallback(
    (event: KeyboardEvent) => {
      const { repeat, ctrlKey, key, metaKey } = event;
```
Does this work @harishfalco ?

Reply:
yes